tests/end2end/cypress/integration/cmdline-wmts-ghaction.js

```diff
@@ -0,0 +1,256 @@
+            .should('contain', 'End generation')
+
+        // Check the cache
+        cy.exec('./../lizmap-ctl docker-exec find /tmp/testsrepository/cache/Quartiers/EPSG_3857/lizmap_/ -type f | wc -l')
+            .its('stdout')
+            .should('contain', '228')
+
+        // Go to the map
+        cy.visit('/index.php/view/map/?repository=testsrepository&project=cache&bbox=425367.933686%2C5402026.205439%2C437311.219354%2C5408953.311127&crs=EPSG%3A3857')
+
+        cy.wait(1000)
+
+        // Check the cache
+        cy.exec('./../lizmap-ctl docker-exec find /tmp/testsrepository/cache/Quartiers/EPSG_3857/lizmap_/ -type f | wc -l')
+            .its('stdout')
+            .should('contain', '228')
+
+        // Login as admin
+        cy.visit('admin.php/auth/login')
+
+        cy.wait(1000)
+
+        cy.get('#jforms_jcommunity_login_auth_login').type('admin')
+        cy.get('#jforms_jcommunity_login_auth_password').type('admin')
+        cy.get('form').submit()
+
+        cy.wait(1000)
+
+        // Go to the map
+        cy.visit('/index.php/view/map/?repository=testsrepository&project=cache&bbox=425367.933686%2C5402026.205439%2C437311.219354%2C5408953.311127&crs=EPSG%3A3857')
+
+        cy.wait(1000)
+
+        // Check the cache
+        cy.exec('./../lizmap-ctl docker-exec find /tmp/testsrepository/cache/Quartiers/EPSG_3857/lizmap_/ -type f | wc -l')
+            .its('stdout')
+            .should('contain', '228')
+
+        // Logout
+        cy.visit('admin.php/auth/login/out')
+
+        cy.wait(1000)
+
+        // Login as user_in_group_a
+        cy.visit('admin.php/auth/login')
+
+        cy.wait(1000)
+
+        cy.get('#jforms_jcommunity_login_auth_login').type('user_in_group_a')
+        cy.get('#jforms_jcommunity_login_auth_password').type('admin')
+        cy.get('form').submit()
+
+        cy.wait(1000)
+
+        // Go to the map
+        cy.visit('/index.php/view/map/?repository=testsrepository&project=cache&bbox=425367.933686%2C5402026.205439%2C437311.219354%2C5408953.311127&crs=EPSG%3A3857')
+
+        cy.wait(1000)
+
+        // Check the cache
+        cy.exec('./../lizmap-ctl docker-exec find /tmp/testsrepository/cache/Quartiers/EPSG_3857/lizmap_/ -type f | wc -l')
+            .its('stdout')
+            .should('contain', '228')
+
+        // Logout
+        cy.visit('admin.php/auth/login/out')
+
+        cy.wait(1000)
+
+        cy.visit('index.php')
+
+        // Cleaning the cache
+        cy.exec('./../lizmap-ctl script lizmap~wmts:cleaning -v testsrepository cache Quartiers', {failOnNonZeroExit: false})
+            .its('stdout')
+            .should('contain', 'Start cleaning')
+
+        // Check the cache
+        cy.exec('./../lizmap-ctl docker-exec find /tmp/testsrepository/cache/Quartiers/EPSG_3857/lizmap_/ -type f | wc -l')
+            .its('stdout')
+            .should('contain', '0')
+    })
+
+    it('lizmap~wmts:capabilities failed', function () {
+        // Only run on local
```

Comment:
Can you write the reason why only local ?

Instead of using the environment variable, you should put it in a new file `tests/end2end/cypress/integration/cmdline-wmts-wrong-cmd.js`, as soon as your PR is merged, I will allow this test to run on GH ;-)